solana-programs/anchor/audius-data/programs/audius-data/src/lib.rs

```diff
@@ -509,12 +528,22 @@ pub struct DeletePlaylist<'info> {
     pub payer: Signer<'info>,
 }
 
+/// Instruction container for verifying a user
+/// Removes playlist storage account entirely
+#[derive(Accounts)]
+pub struct VerifyUser<'info> {
+    pub audius_admin: Account<'info, AudiusAdmin>,
+    pub user: Account<'info, User>,
+    pub authenticator: Signer<'info>,
```

Comment:
can we require this to be a signer w/a constraint? then verify membership inside